src/utils/__tests__/Throttler.test.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+import Throttler from '../Throttler';
+
+describe('features/targeting/utils/makeThrottle', () => {
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    test('throttle if the one parameter is passed', async () => {
+        const callback = jest.fn().mockImplementation(x => x + 1);
+        const throttler = new Throttler();
+        expect(await throttler.throttle(() => callback(1), 1000, [1])).toEqual(2);
+        expect(await throttler.throttle(() => callback(1), 1000, [1])).toEqual(2);
+        expect(callback.mock.calls).toEqual([[1]]);
+        expect(await throttler.throttle(() => callback(3), 1000, [2])).toEqual(4);
+        expect(callback.mock.calls).toEqual([[1], [3]]);
+    });
+
+    test('will throttle within expiration date', async () => {
```

Comment:
Minor: I'd recommend editing the names of these tests to match the other tests in this repo (e.g., "**should** throttle within expiration date")